spellbook/data_formatting/serialize.py

```diff
@@ -27,18 +27,18 @@ def convert_string(string):
         return string
 
 
-def nested_dict(var_list, splitter="/"):
+def nested_dict(var_list, splitter="/", delimiter="="):
     output = {}
     for v in var_list:
-        keys, val = v.split("=")
+        keys, val = v.split(delimiter)
```

Comment:
Is it worth making the fix to only grab the first entry as the key and everything else as the values?

Reply:
Sure, let me update the request.